frontend/src/pages/projects/screens/detail/notebooks/NotebookTableRow.tsx

```diff
@@ -62,7 +65,11 @@ const NotebookTableRow: React.FC<NotebookTableRowProps> = ({
       requests: {},
     },
   };
+  const { dashboardNamespace } = useNamespaces();
+  const [images, imagesLoaded, imagesLoadError] = useImageStreams(dashboardNamespace, true);
```

Comment:
for each notebook we are fetching all the imagestreams, this is a lot of extra calls. we should try to not fetch all theses images until we enter the modal to reduce the number of calls

Reply:
or we should upgrade useNotebookImage becuase that is already fetching all images so look at that route as well